external/js/cothority/tsconfig.json

```diff
@@ -14,7 +14,17 @@
     "typeRoots": [
       "../types",
       "./node_modules/@types"
-    ]
+    ],
+    "paths": {
+      "@dedis/kyber": [
+        "node_modules/@dedis/kyber/dist",
+        "node_modules/@dedis/kyber"
+      ],
+      "@dedis/kyber/*": [
+        "node_modules/@dedis/kyber/dist/*",
+        "node_modules/@dedis/kyber/*"
+      ]
+    }
```

Comment:
debug? should you use `npm run link` in kyber to do so?

Reply:
It's for debugging, but I thought to leave it there. The problem with `npm link` is that it points to the root-directory of the package. And the compiled code is in the `dist` directory. So this searches first in the `dist` directory, and only then in the actual package.

The first is used with `npm link`, the second with a 'normal' `npm ci` install.

It is the most clean way to use `npm link` with typescript, IMO.